ui/src/app/cohort-search/attributes-page/attributes-page.component.tsx

```diff
@@ -473,6 +473,9 @@ export const AttributesPage = fp.flow(
             } else {
               // CAT attributes are displayed as checkboxes in the attributes form
               if (parseInt(attr.estCount, 10) > 0) {
+                // Property 'checked' does not exist on type 'CriteriaAttribute'.
+                // TODO RW-5572 confirm proper behavior and fix
+                // eslint-disable-next-line @typescript-eslint/dot-notation
```

Comment:
Need to disable the rule in 10 instances.  Some of these may be intended, but some may be bugs.  Let's investigate these.